src/utils/blockchain/contracts.testnet.ts

```diff
@@ -30,6 +31,15 @@ export const contracts = {
     abi: bzxAbi,
     blockNumber: 1218686,
   },
+  sovToken: {
+    address: '0x6a9A07972D07e58F0daf5122d11E069288A375fb',
+    abi: SovTokenABI,
+    blockNumber: 2742422,
+  },
+  vestingRegistry2: {
+    address: '0x310006E356b0818C3Eaf86a9B2f13013d4691a1c',
+    abi: VestingRegistryAbi as any,
+  },
```

Comment:
SOV token contract is already defined and called `SOV_token`, you will need to use this name from now one.
Vesting registry 2 is called `vestingRegistryOrigin`

```suggestion
```

Reply:
done